gnome-image-installer/pages/diskimage/gis-diskimage-page.c

```diff
@@ -103,7 +103,7 @@ gis_diskimage_page_selection_changed(GtkWidget *combo, GisPage *page)
   gis_store_set_object (GIS_STORE_IMAGE, G_OBJECT (file));
   if (g_str_has_suffix (image, ".gz"))
     {
-      gint64 size = get_gzip_disk_image_size (image);
+      guint64 size = get_gzip_disk_image_size (image);
       if (size <= 0)
```

Comment:
These `size <= 0` comparisons can be clarified as `size == 0` now (in several places below as well).